src/EditorFeatures/CSharpTest/MisplacedUsingDirectives/MisplacedUsingDirectivesInCompilationUnitCodeFixProviderTests.cs

```diff
@@ -0,0 +1,488 @@
+                Diagnostic(MisplacedUsingDirectivesInCompilationDiagnosticAnalyzer.InsideDescriptor).WithLocation(4, 1),
+                Diagnostic(MisplacedUsingDirectivesInCompilationDiagnosticAnalyzer.InsideDescriptor).WithLocation(6, 1),
+                Diagnostic(MisplacedUsingDirectivesInCompilationDiagnosticAnalyzer.InsideDescriptor).WithLocation(7, 1),
+                Diagnostic(MisplacedUsingDirectivesInCompilationDiagnosticAnalyzer.InsideDescriptor).WithLocation(9, 1),
+                Diagnostic(MisplacedUsingDirectivesInCompilationDiagnosticAnalyzer.InsideDescriptor).WithLocation(10, 1),
+            };
+
+            return VerifyCodeFixAsync(testCode, InsideNamespaceOption, expected, fixedTestCode, placeSystemNamespaceFirst: true);
+        }
+
+        /// <summary>
+        /// Verifies that the code fix will move the using directives and not sort them alphabetically.
+        /// </summary>
+        [Fact]
+        public Task WhenInsidePreferred_UsingsInCompilationUnit_UsingsAndWithAlphaSortIgnored()
+        {
+            var testCode = @"using Microsoft.CodeAnalysis;
+using SystemAction = System.Action;
+using static System.Math;
+using System;
+
+using static System.String;
+using MyFunc = System.Func<int, bool>;
+
+using System.Collections.Generic;
+using System.Collections;
+
+namespace NamespaceName
+{
+    public class Bar
+    {
+    }
+}
+";
+
+            var fixedTestCode = @"namespace NamespaceName
+{
+    using Microsoft.CodeAnalysis;
+    using SystemAction = System.Action;
+    using static System.Math;
+    using System;
+
+    using static System.String;
+    using MyFunc = System.Func<int, bool>;
+
+    using System.Collections.Generic;
+    using System.Collections;
+
+    public class Bar
+    {
+    }
+}
+";
+
+            var expected = new DiagnosticResult[]
+            {
+                Diagnostic(MisplacedUsingDirectivesInCompilationDiagnosticAnalyzer.InsideDescriptor).WithLocation(1, 1),
+                Diagnostic(MisplacedUsingDirectivesInCompilationDiagnosticAnalyzer.InsideDescriptor).WithLocation(2, 1),
+                Diagnostic(MisplacedUsingDirectivesInCompilationDiagnosticAnalyzer.InsideDescriptor).WithLocation(3, 1),
+                Diagnostic(MisplacedUsingDirectivesInCompilationDiagnosticAnalyzer.InsideDescriptor).WithLocation(4, 1),
+                Diagnostic(MisplacedUsingDirectivesInCompilationDiagnosticAnalyzer.InsideDescriptor).WithLocation(6, 1),
+                Diagnostic(MisplacedUsingDirectivesInCompilationDiagnosticAnalyzer.InsideDescriptor).WithLocation(7, 1),
+                Diagnostic(MisplacedUsingDirectivesInCompilationDiagnosticAnalyzer.InsideDescriptor).WithLocation(9, 1),
+                Diagnostic(MisplacedUsingDirectivesInCompilationDiagnosticAnalyzer.InsideDescriptor).WithLocation(10, 1),
+            };
+
+            return VerifyCodeFixAsync(testCode, InsideNamespaceOption, expected, fixedTestCode, placeSystemNamespaceFirst: false);
```

Comment:
skipping rest of tests for now, but i don't know why tehse aren't following the patterns and practices of the rest of the IDE tests.

Reply:
Updated.